src/main/java/com/mx/iwaconsolti/calculator/MultiplicationService.java

```diff
@@ -0,0 +1,17 @@
+package com.mx.iwaconsolti.calculator;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+@Getter
+@Setter
+@Component
+public class MultiplicationService {
+    private double num1;
```

Comment:
This works, but in real life, is not recommended that you put variables which value can change. Since the beans/components are only created once, and then that bean is used everytime it is injected, if two users are doing an operation they can override the value for the other person.

That's why we need to make the beans stateless class, so they cannot have wierd behaviour when used by multiple users at the same time